packages/nimbus/src/components/collapsible-motion/components/collapsible-motion-content.tsx

```diff
@@ -0,0 +1,31 @@
+import { forwardRef, useRef } from "react";
+import { useObjectRef } from "react-aria";
+import { mergeRefs } from "@chakra-ui/react";
+import { useCollapsibleMotionContext } from "./collapsible-motion-context";
+import { CollapsibleMotionContentSlot } from "../collapsible-motion.slots";
+import type { CollapsibleMotionContentProps } from "../collapsible-motion.types";
+
+/**
+ * CollapsibleMotion.Content - The collapsible content container
+ *
+ * This component renders the content that will expand and collapse.
+ * It handles accessibility attributes through React Aria.
+ */
+export const CollapsibleMotionContent = forwardRef<
+  HTMLDivElement,
+  CollapsibleMotionContentProps
+>(({ children, ...props }, forwardedRef) => {
+  const { panelProps, panelRef } = useCollapsibleMotionContext();
+
+  // Create a local ref and merge with both panelRef (for React Aria) and forwardedRef (for user)
+  const localRef = useRef<HTMLDivElement>(null);
+  const ref = useObjectRef(mergeRefs(localRef, panelRef, forwardedRef));
+
+  return (
+    <CollapsibleMotionContentSlot ref={ref} {...panelProps} {...props} asChild>
```

Comment:
if the consumer passes an `id` to the content slot, it will overwrite the `id` from context, and result in the aria attributes being misconfigured in the DOM, should the spreads here be reversed? Or should the context be handled differently to account for the props passed to the compound components?